README.module_tester.js

```diff
@@ -0,0 +1,72 @@
+// Copyright 2023 Cisco Systems, Inc.
```

Comment:
I don't want another test file in root of project. README.test.js is bad enough.